mutag_calib/configs/pt_reweighting/ptreweighting_run3.py

```diff
@@ -124,11 +124,9 @@
 
     preselections = [get_nObj_min(1, parameters.object_preselection["FatJet"]["pt"], "FatJetGood")],
     categories = {
-        "inclusive" : [passthrough],
```

Comment:
Getting an error from this change, I think: https://gist.github.com/GNiendorf/9f5d8096a544ebf66d82eed7bfd5aa99

Reply:
It's due to the `cat = inclusive` here:

https://github.com/mmarchegiani/mutag-calib/blob/main/mutag_calib/workflows/mutag_processor.py

At line 55.

The category inclusive has been removed because now the recommendation is to apply the 3D reweighting derived in the region and of the analysis, i.e. pt300msd30to210 for the HHbbgg analysis.

I will push a fix soon to make this category a parameter in mutag_calibration.yaml